apis/apps/v1alpha1/type.go

```diff
@@ -243,13 +243,12 @@ const (
 
 // HScaleDataClonePolicyType defines data clone policy when horizontal scaling.
 // +enum
-// +kubebuilder:validation:Enum={None,Snapshot,Backup}
+// +kubebuilder:validation:Enum={None,Backup}
 type HScaleDataClonePolicyType string
 
 const (
-	HScaleDataClonePolicyNone         HScaleDataClonePolicyType = "None"
-	HScaleDataClonePolicyFromSnapshot HScaleDataClonePolicyType = "Snapshot"
```

Comment:
do explain why an improvement is by remove an exiting function?

Reply:
add back as deprecated